test/e2e-helm/e2e_helm_olm_test.go

```diff
@@ -44,16 +43,18 @@ var _ = Describe("Integrating Helm Projects with OLM", func() {
 			By("building the operator bundle image")
 			// Use the existing image tag but with a "-bundle" suffix.
 			imageSplit := strings.SplitN(tc.ImageName, ":", 2)
-			bundleImage := path.Join("quay.io", imageSplit[0]+"-bundle")
+			bundleImage := imageSplit[0] + "-bundle"
 			if len(imageSplit) == 2 {
 				bundleImage += ":" + imageSplit[1]
 			}
 			err = tc.Make("bundle-build", "BUNDLE_IMG="+bundleImage)
 			Expect(err).NotTo(HaveOccurred())
 
-			By("loading the project image into Kind cluster")
-			err = tc.LoadImageToKindClusterWithName(bundleImage)
-			Expect(err).Should(Succeed())
+			if isRunningOnKind() {
+				By("loading the bundle image into Kind cluster")
+				err = tc.LoadImageToKindClusterWithName(bundleImage)
+				Expect(err).Should(Succeed())
```

Comment:
Nit: I know this is a backport and this line is basically unchanged in this PR but for better readability this probably should be:
```Go
Expect(err).NotTo(HaveOccurred())
```
`Succeed()` is used more for the function itself:
```Go
Ω(DoSomethingSimple()).Should(Succeed())
```
https://onsi.github.io/gomega/#handling-errors

Reply:
+1 I'll make this comment into an issue. There are probably lots of instances of this that we can fix on master (probably not worth backporting those changes though).